internal/controllers/core/tiltfile/reconciler.go

```diff
@@ -426,6 +426,12 @@ func DecideRegistry(ctx context.Context, kCli k8s.Client, tlr *tiltfile.Tiltfile
 		// If we've found a local registry in the cluster at run-time, use that
 		// instead of the default_registry (if any) declared in the Tiltfile
 		logger.Get(ctx).Infof("Auto-detected local registry from environment: %s", registry)
+
+		if !tlr.DefaultRegistry.Empty() {
+			// The user has specified a default registry in their Tiltfile, but it will be ignored.
+			logger.Get(ctx).Warnf("Default registry specified, but will be ignored in favor of auto-detected registry.")
```

Comment:
this should be an info log rather than a warning log

warnings mean "this code is incorrect 100% of the time and we can't do what you want here"

There are legitimate reasons to specify `default_registry` but still prefer a local cluster, as described in this example: https://docs.tilt.dev/personal_registry.html#the-medium-easy-way-for-4-of-users

Reply:
Changed.

For an individual, setting default registry never has an effect if they have a local registry. So it can still be useful for a team to have it in their Tiltfile for cases where some individuals may not have a cluster with a local registry?

Reply:
yep! many teams use both remote clusters and local clusters depending on what they're doing.